galaxy.yml

```diff
@@ -1,20 +1,20 @@
 namespace: community
 name: windows
-version: 0.0.1-beta.1
+version: 0.0.1
```

Comment:
I would prefer to do a beta release first when the 2.10 beta is out and only do the non-preview release when 2.10 is out officially.

Reply:
Ah, I'd already published this to Galaxy. We needed to get a full complement of collections out there.

People understand that for `0.` that these aren't for production use.

Reply:
Ah, I'd already published this to Galaxy. We needed to get a full complement of collections out there.

People understand that for `0.` that these aren't for production use.

Reply:
If you've already published that's fine, it just means that we now need to handle version_added, changelogs, and all other release based stuff which isn't really in place right now.

Reply:
There's no expectation that `0.y.x` releases on Galaxy are in any way correct or usable. So I hope by my publishing what we had yesterday doesn't impact anyone.